### ISSUE TITLE
patch 8.2

### ISSUE THREAD

**Issue by knudsendk123** [TimoshN/FloatIndicators#3]
will you be updateing this addon for 8.2 ? the biggest use for me for this addon is the name plates for m+
so would realy love it if you are able to keep it updated

**Comment by TimoshN** [comment]
no

**Comment by Albrod** [comment]
can you say why don't want to keep this up to date ? I'm considering continuing your work if you don't want to keep going on.

**Comment by TimoshN** [comment]
With new anchor system, blizzard remove methode i get nameplate position on screen + they fix secure env and now you cant see forbidden frames ( friendly nameplates )

**Comment by EKE00372** [comment]
Hi TimoshN, classic wow based on 8.01 API so forbidden nameplates seems can work on 1.13.2. do you have plan for it ?_?

**Comment by TimoshN** [comment]
No. In classic nameplates distance limited to 20y. it pointless
